internal/graph/resolvers/composite_test.go

```diff
@@ -438,6 +473,33 @@ func TestCompositeResourceSpecClaim(t *testing.T) {
 	}
 }
 
+func TestCompositeResourceSpecClaimReference(t *testing.T) {
+	s := &compositeResourceSpec{}
+	ctx := graphql.WithResponseContext(context.Background(), graphql.DefaultErrorPresenter, graphql.DefaultRecover)
+
+	got, err := s.ClaimReference(ctx, &model.CompositeResourceSpec{ClaimReference: &corev1.ObjectReference{Name: "some ref"}})
+	errs := graphql.GetErrors(ctx)
+
+	AssertNoErrors(t, err, errs)
+
+	name := "some ref"
+	want := &model.ObjectReference{Name: &name}
+	if diff := cmp.Diff(want, got, test.EquateErrors()); diff != "" {
+		t.Errorf("s.ClaimReference(...): -want, +got:\n%s\n", diff)
+	}
+
+	ctx = graphql.WithResponseContext(context.Background(), graphql.DefaultErrorPresenter, graphql.DefaultRecover)
+	got, err = s.ClaimReference(ctx, &model.CompositeResourceSpec{ClaimReference: nil})
+	errs = graphql.GetErrors(ctx)
+
+	AssertNoErrors(t, err, errs)
+
+	want = nil
+	if diff := cmp.Diff(want, got, test.EquateErrors()); diff != "" {
+		t.Errorf("s.ClaimReference(...): -want, +got:\n%s\n", diff)
+	}
```

Comment:
I notice these tests aren't following the table driven conventions of the existing tests - what's the reason for that?

Reply:
There were only two cases and I felt the overhead for making it a table was more costly than just repeating. Is it better to just stay consistent you think?

Reply:
I would prefer to stay consistent - especially if it's possible we might add other cases in future.